### PR TITLE
[code] don't report empty vscode extensions as error

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

We have many false positive errors for repos without VS Code extensions in .gitpod.yml [[1](https://console.cloud.google.com/errors/detail/CNzsobWPtZqyzgE?project=workspace-clusters)] This PR does not treat them as error.

## Related Issue(s)

Fixes #

## How to test

- Switch to latest VS Code: https://ak-code-em8be1e68403.preview.gitpod-dev.com/preferences
- Start a workspace without VS Code extensions: https://ak-code-em8be1e68403.preview.gitpod-dev.com/#https://github.com/akosyakov/parcel-demo
  - It should start without any issues. Check workspace lgos that there is no `config.vscode field not exists` error anymore.
- Start a workspace with VS Code extensions: https://ak-code-em8be1e68403.preview.gitpod-dev.com/#https://github.com/gitpod-io/spring-petclinic
  - It should start without any issues, Java extensions should be installed automatically.

## Release Notes

```release-note
NONE
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

**Comment by andreafalzetti** [comment]
@akosyakov I would like to see this error in prod, where does it appear?

**Review by jeanp413 — APPROVED** [review]
/hold wait for @andreafalzetti review

**Comment by akosyakov** [comment]
@andreafalzetti https://console.cloud.google.com/errors/detail/CNzsobWPtZqyzgE?project=workspace-clusters they are all from prod

**Comment by felladrin** [comment]
But as the workspaces from Preview Environment are not part of _workspace-clusters_, they wouldn't be displayed on logs from https://console.cloud.google.com/logs/query?project=workspace-clusters, right?
(I think preview environment workspaces are part of "gitpod-core-dev" project)

File: components/ide/code/codehelper/main.go [hunk 1]
```diff
@@ -166,7 +166,6 @@ func getExtensions(repoRoot string) (extensions []Extension, err error) {
 		return
 	}
 	if config == nil || config.Vscode == nil {
-		err = errors.New("config.vscode field not exists")
```
Comment (felladrin) [inline components/ide/code/codehelper/main.go:169]: **Note (non-blocking):** For the _spring-petclinic_ repository I got this warning:
image

**Review by felladrin — APPROVED** [review]

**Comment by akosyakov** [comment]
> But as the workspaces from Preview Environment are not part of workspace-clusters, they wouldn't be displayed on logs from https://console.cloud.google.com/logs/query?project=workspace-clusters, right?

ah yes, sorry, you need to start dev workspace on this branch and use kubectl logs for your workspace instance id

File: components/ide/code/codehelper/main.go (hunk 1 above)
Comment (akosyakov) [reply components/ide/code/codehelper/main.go:169]: I think it is known bug for lang packs cc @jeanp413

Comment (jeanp413) [reply components/ide/code/codehelper/main.go:169]: yeah different issue

**Comment by akosyakov** [comment]
@andreafalzetti is it fine with you to merge?

**Review by andreafalzetti — APPROVED** [review]

**Pull request merged**